src/parse.rs

```diff
@@ -870,34 +870,29 @@ where
             }
             match u16::from_str_radix(&s, 16) {
                 Ok(u) => {
-                    if (0xDC00..=0xDFFF).contains(&u) || (0xD800..=0xDB7F).contains(&u) {
-                        // Low/High Surrogates
+                    if (0xD800..=0xDBFF).contains(&u) {
+                        // Low Surrogates
```

Comment:
These are actually the high surrogates 🤷

Reply:
Ah, my comment was to signal that we're trying to parse the low surrogates next 😅
I'll add a more descriptive comment